UnityProjects/MRTKDevTemplate/Assets/Scripts/SpeechRecognitionHandler.cs

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.MixedReality.Toolkit.Subsystems;
+using UnityEngine;
+using UnityEngine.Events;
+
+namespace Microsoft.MixedReality.Toolkit.Examples.Demos
+{
+    /// <summary>
+    /// Demonstration script showing how to subscribe to and handle
+    /// events fired by SpeechRecognitionSubsystem.
+    /// </summary>
+    [AddComponentMenu("MRTK/Examples/Speech Recognition Handler")]
+    public class SpeechRecognitionHandler : MonoBehaviour
```

Comment:
Can we make it internal just so folks don't end up taking a dep on it?

Reply:
This script is a sample script (i.e. in Assembly-CSharp) so not sure if `internal`ing it would do much.